### PR TITLE
use hourglass icon and update time in menu

### PR DESCRIPTION
**Pull request by tinnet**
added hourglass icons to the menubar

removed the blinking dots and only show time down to minutes because when the menu is open the app does not tick anymore, so we can't update the title

### CODE REVIEW

File: desktime/StatusMenuController.swift [hunk 1]
```diff
@@ -103,16 +103,18 @@ class StatusMenuController: NSObject {
             lastDayOfYear = currentDayOfYear
             runTime = 0;
         }
-    
+        NSLog("lala")
```
Comment (michaelcontento) [inline desktime/StatusMenuController.swift]: lala?

Comment (tinnet) [inline desktime/StatusMenuController.swift]: log line that shows that the app stops when the menu is open ;)

File: desktime/StatusMenuController.swift [hunk 2]
```diff
@@ -122,10 +124,14 @@ class StatusMenuController: NSObject {
...
-        statusItem.title = (isOverLimit ? "!! " : "")
-            + String(format:"%02d", hours)
+
+        timeDisplay.title = String(format:"%02d", hours)
             + (showDots ? ":" : " ")
-            + String(format:"%02.f", minutes);
+            + String(format:"%02.f", minutes)
+            + (showDots ? ":" : " ");
+//            + String(format:"%02.f", seconds);
+        timeDisplay.title = String(format:"%d Hours %.f", hours, minutes);
```
Comment (michaelcontento) [inline desktime/StatusMenuController.swift]: please drop variant

**Comment by tinnet** [comment]
feedback applied, ready to merge i think

**Pull request merged**